pufferlib/ocean/trash_pickup/trash_pickup.c

```diff
@@ -0,0 +1,74 @@
+#include <time.h>
+#include "trash_pickup.h"
+
+#define INDEX_C(env, x, y) ((y) * (env).grid_size + (x))
+
+// Demo function for visualizing the TrashPickupEnv
+
+
+void demo(int grid_size, int num_agents, int num_trash, int num_bins, int max_steps) {
+    CTrashPickupEnv env;
+    env.grid_size = grid_size;
+    env.num_agents = num_agents;
+    env.num_trash = num_trash;
+    env.num_bins = num_bins;
+    env.max_steps = max_steps;
+
+    allocate(&env);
+
+    Client* client = make_client(&env);
+
+    reset(&env);
+
+    while (!WindowShouldClose()) {
+        // Random actions for all agents
+        for (int i = 0; i < env.num_agents; i++) {
```

Comment:
Missing a human control mode. See other envs for how to do that. You want human to be able to hold shift and override 1 agent

Reply:
Added.